tokenomics.md

```diff
@@ -27,23 +27,21 @@ Token allocation is benchmarked to similar successful projects. Typically, such
 
-It is thus important for the OBX/ETH price to remain as stable as possible. The incentives of the various groups involved in the network should be able to facilitate this. To participate in the network or become an aggregator, users must buy OBX tokens. On the flip side, aggregators will likely sell OBX to cover their ETH expenses. These flows coupled with large amounts of OBX locked up in staking arrangements should enable OBX to retain enough value versus ETH to ensure being an aggregator is profitable.
+It is thus essential for the OBX/ETH price to remain as stable as possible. The incentives of the various groups involved in the network should be able to facilitate this. To participate in the network or become an aggregator, users must buy OBX tokens. On the flip side, aggregators are likely to sell OBX to cover their ETH expenses. These flows coupled with large amounts of OBX locked up in staking arrangements should enable OBX to retain enough value versus ETH to ensure being an aggregator is profitable.
 
-Gas will be accounted for in the same way that Ethereum does and there will also have a market for OBX gas based upon the supply and demand determined by Obscuro node operators/verifiers. Clearly, income of OBX _in ETH terms_ to an aggregator must be greater than the ETH they require to publish the block to Ethereum Mainnet.
+Gas is accounted for in the same way Ethereum does, with a market for OBX gas based upon the supply and demand determined by Obscuro node operators/verifiers. Income of OBX _in ETH terms_ to an aggregator must be greater than the ETH they require to publish the block to Ethereum Mainnet.
 
 ## Illustrated flows
 ![token-flow](./images/token-flow.png)
 
-Over time the community pool decreases in size. The eco-system pool also decreases in size. The amount of tokens held by the community (users, market makers, witnesses and node operators) increases in size. Likewise, the team pool will likely decrease in size over time as some tokens are sold to fund development.
+Over time the community pool decreases in size. Likewise, the eco-system pool also decreases in size. However, the amount of tokens held by the community (users, market makers, witnesses and node operators) increases in size. Likewise, the team pool decreases in size over time as tokens are sold to fund development.
 
-[^1]: OBX is not a security because there is no expectation of profit, indeed, OBX  will be issuer-less in that no incorporated entity credits their balance sheet equity or liabilities in conjunction with the issuance of OBX. OBX is not a regular coin as it is not intended as a generic medium of exchange.
\ No newline at end of file
+[^1]: OBX is not a security because there is no expectation of profit. Indeed, OBX  is issuer-less in that no incorporated entity credits its balance sheet equity or liabilities in conjunction with the issuance of OBX. In addition, OBX is not a regular coin as it is not for use as a generic medium of exchange.
```

Comment:
Leslie thinks lines like this will inflame regulators. Not sure it adds value. Remove it.